### ISSUE TITLE
[Edit] CSS [Edit] CSS Background: background-image

### ISSUE THREAD

**Issue by codecademy-docs** [Codecademy/docs#3535]
### Type of Edit (select all that apply)

Add new content (definitions, codeblocks, etc.)

### Description (optional)

We would like an edit a term within the CSS Animations entry. The entry lives in docs/content/css/concepts/animations/terms/background-image/background-image.md

Please refer to the [term entry templates](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

Please:
1) Add supplemental media (preferred .png or .gif, <1MB).
2) Media (images, gifs) should accurately represent the existing code blocks in the example sections and fit in-line.

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [x] Labels added
- [ ] Issue is assigned

**Comment by Atul72** [comment]
Heya @bndiep , can i work on this issue?

**Comment by luv2027** [comment]
Heya @bndiep , can i work on this issue?

**Comment by yangc95** [comment]
@Atul72 @luv2027 See [maintainers here](https://github.com/Codecademy/docs#-content-team)

This will be assigned to @Atul72 since we assign issues on a first-come-first-serve basis. Thanks for understanding this process.

Atul72 You're assigned! You can check out the [Contribution Guide](<https://github.com/Codecademy/docs/blob/main/.github/CONTRIBUTING.md>) for additional information on contributing. 
 
See [Content Standards](<https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md>) for an overview of styling tips that the [Style Guide](<https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md>) covers in detail. 
 
After creating a PR, maintainer(s) (with the collaborator label) will add in comments/suggestions to address any revisions prior to approving. 
 
What brought you to Docs?

**Comment by luv2027** [comment]
@yangc95  I did not know that it was first come first serve basis. I thought It was open for all so i created file. Btw thank you for reviewing my PR. I hope to contribute more to this repo.

**Comment by yangc95** [comment]
> @yangc95  I did not know that it was first come first serve basis. I thought It was open for all so i created file. Btw thank you for reviewing my PR. I hope to contribute more to this repo.

@luv2027 Yes! Issues are open for all in the sense that contributors with good-standing can ask to be assigned to an unclaimed issue. However that doesn't guarantee you'll be assigned to that issue in cases like that above (another contributor has asked to be assigned prior).

No problem for the review 🙂

**Comment by SSwiniarski** [comment]
@Atul72 It's been two weeks, are you still working on this entry? Note that in another week it will be unassigned due to inactivity.

**Comment by Atul72** [comment]
@SSwiniarski , Sorry I can't do the task. I am suffering from dengue. It's been two weeks. My health was still low. I am unassigning myself. Again sorry.

**Comment by scottfinamore** [comment]
I am looking to work on my first issue for a codecademy project. If I may, I would like to work on this.

**Comment by SSwiniarski** [comment]
@scottfinamore You're assigned! You can check out the [Contribution Guide](https://github.com/Codecademy/docs/blob/main/.github/CONTRIBUTING.md) for additional information on contributing.
See [Content Standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md) for an overview of styling tips that the [Style Guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md) covers in detail.
After creating a PR, maintainer(s) (with the collaborator label) will add in comments/suggestions to address any revisions prior to approving.
	
In addition, please head over to the [Docs Forum](https://discuss.codecademy.com/c/community/community-docs/2205) if you have any questions about Docs or reply to the thread on [Possible Content Discussion](https://discuss.codecademy.com/t/find-a-starting-point-possible-content-discussion/745868) to share ideas and collaborate with other contributors, maintainers, or super users to determine good future issues that will help expand Docs!

If this is your first time contributing, can you let us know how you came across this repo?

**Comment by SSwiniarski** [comment]
@scottfinamore It's been two weeks. Are you still working on this entry? Note that in another week it will be unassigned due to inactivity.

**Comment by scottfinamore** [comment]
Yes I will still fix this issue. I apologize some work stuff came up and stole my attention. Im happy to work on it today as I'm free.

**Comment by SSwiniarski** [comment]
@scottfinamore this is being unassigned due to inactivity.

**Comment by Brag2gr8** [comment]
Kindly assign this to me

**Comment by SSwiniarski** [comment]
@Brag2gr8 You're assigned! You can check out the [Contribution Guide](https://github.com/Codecademy/docs/blob/main/CONTRIBUTING.md) for additional information on contributing.
See [Content Standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md) for an overview of styling tips that the [Style Guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md) covers in detail.
After creating a PR, maintainer(s) (with the collaborator label) will add in comments/suggestions to address any revisions prior to approving.
	
In addition, please head over to the [Docs Forum](https://discuss.codecademy.com/c/community/community-docs/2205) if you have any questions about Docs or reply to the thread on [Possible Content Discussion](https://discuss.codecademy.com/t/find-a-starting-point-possible-content-discussion/745868) to share ideas and collaborate with other contributors, maintainers, or super users to determine good future issues that will help expand Docs!

If this is your first time contributing, can you let us know how you came across this repo?
